app/models/concerns/taggable.rb

```diff
@@ -80,6 +77,9 @@ def freeform_string=(tag_string)
     parse_tags(Freeform, tag_string)
   end
 
+  alias category_strings= category_string=
```

Comment:
The plural form also makes it much simpler to use `collection_check_boxes`. Because `category_strings` returns an array of tag names (which `collection_check_boxes` can use to figure out which items to check by default), whereas `category_string` returns the same array joined by commas (which `collection_check_boxes` doesn't understand).

Reply:
I should have said `category_string=` (setter). We already have 2 versions of the getter returning different formats (the plural one is better for checkboxes, as you said). The setter can handle either string or array arguments, so the only reason to add a setter alias is for symmetry/readability, right?

Reply:
Passing the plural to the checkboxes means that it's automatically used as both the getter name and the setter name. I wanted the simplicity from the plural getter, so I had to add the corresponding plural setter.